WordPress/src/main/java/org/wordpress/android/util/extensions/CompatExtensions.kt

```diff
@@ -18,3 +23,55 @@ fun OnBackPressedDispatcher.onBackPressedCompat(onBackPressedCallback: OnBackPre
     onBackPressed()
     onBackPressedCallback.isEnabled = true
 }
+
+/**
+ * TODO: Remove this when stable androidx.core 1.10 is released. Use IntentCompat instead.
+ */
+@Suppress("ForbiddenComment")
+inline fun <reified T : Parcelable> Intent.getParcelableExtraCompat(key: String): T? =
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
+        getParcelableExtra(key, T::class.java)
+    } else {
+        @Suppress("DEPRECATION")
+        getParcelableExtra(key) as T?
+    }
+
+inline fun <reified T : Serializable> Intent.getSerializableExtraCompat(key: String): T? =
```

Comment:
Actually, that was intentional. They won't include a compatibility function for those functions. You can check the source code of [IntentCompat](https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:core/core/src/main/java/androidx/core/content/IntentCompat.java) and [BundleCompat](https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:core/core/src/main/java/androidx/core/os/BundleCompat.java).

Reply:
Oh, this is good to know, thanks for noticing! 🙇 ❤️ 

Suggestion (💡): Maybe then we could add a KDoc to explain exactly that so that next time someone like me won't ask the same question twice, wdyt? 🤔

Reply:
> Suggestion (💡): Maybe then we could add a KDoc to explain exactly that so that next time someone like me won't ask the same question twice, wdyt? 🤔

Done: 4c32fd4d100884124bfe6df69ef915c3a4ff0311

Reply:
Love it, thanks @irfano ! 🙇 ❤️